src/Recursion.java

```diff
@@ -0,0 +1,33 @@
+/* Написать функцию, которая проверяет, является ли слово палиндромом */
+
+public class Recursion {
+    //Первый способ
+    public static boolean rec(String str, int n) {
+        int l = str.length();
+        if ((n < l / 2) && (str.charAt(n) == str.charAt(l - n - 1)) && (l > 1)) {
+            return rec(str, n + 1);
+        } else if (n >= l / 2) {
```

Comment:
[ERROR] /Users/geoiva/Desktop/SberJava/students/maxim_rusalev/HomeWork/src/Recursion.java:9:16: Conditional logic can be removed. [SimplifyBooleanReturn]